Source/OxyPlot/Utilities/TypeExtensions.cs

```diff
@@ -26,12 +26,20 @@ public static class TypeExtensions
         /// <returns>An object that represents the specified property, or null if the property is not found.</returns>
         public static PropertyInfo GetRuntimeProperty(this Type type, string name)
         {
+#if NET40
+            var source = type.GetProperties();
```

Comment:
I think this should be `var source = type.GetProperties(BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static | BindingFlags.Instance);` to be equivalent to `var source = typeInfo.AsType().GetRuntimeProperties();`